.github/workflows/package-audit.yml

```diff
@@ -0,0 +1,22 @@
+name: Package Audit
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+  schedule:
+  - cron: "0 0 1 * *" # every month
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache: npm
+
+    - run: npm audit --omit dev
```

Comment:
Curious, why `--omit dev` vs `--production`? Is there even a difference?

Reply:
I guess reading it [may just be shorthand](https://docs.npmjs.com/cli/v8/commands/npm-audit#bulk-advisory-endpoint):

>either via the --omit config, or one of the shorthands such as --production, --only=dev

Reply:
I believe they're equivalent in this case.